aces/imaging/TP-ACA-feather.py

```diff
@@ -0,0 +1,200 @@
+# This script is to feather 7m and TP cubes.
+
+
+# A template CMZ cube for mosaic
+#templateIM='/almalustre/home/pyhsieh/ESO/spw24/template/CS2-1-temlate-whole-band.image'   # whole band
+#templateIM='/almalustre/home/pyhsieh/ESO/spw24/template/CS2-1-temlate-narrow-band.image'  # only 0~1500 channel
+templateIM='CS2-1-temlate-whole-band.image'
+
+import glob
+linename=glob.glob('*.image.pbcor')
```

Comment:
Let's change this to
```suggestion
linename=glob.glob('*spw33*.image.pbcor')
```
so that, if you run it in a directory with other cubes in it by accident, you don't change their header frequency value unintentionally